### ISSUE TITLE
Comparison with other Neural Networks

### ISSUE THREAD

**Issue by Twice22** [JizhiziLi/GFM#13]
Hey!

Thank you for sharing your work. It is always nice to have insights of STOTA ideas to handle certain computer vision tasks.
This is not really an issue but more like a request. I've noticed, while reading your paper, that you compare your result
to HAttMatting.

I, myself, have reimplemented their papers but my implementations contains 42M parameters, while you are claiming that it contains 109M params. I have used this piece of code

```
pytorch_total_params = sum(p.numel() for p in netG.parameters() if p.requires_grad)
``` 

only on the generator. To compute the number of trainable parameters.

Yet, My HAttMatting neural network is able to output quite precise alpha mask very quickly (due to pretrained model weights)

So my question is the following: can you share the pth/pt file (net the code) of your reimplementation of HAttMatting neural network because it looks a bit weird to me that it contains 109M parameters.

Thanks a lot

**Comment by ghost** [comment]
Hi @Twice22, would you mind sharing your implementation of HAttMatting. I started implementing the same but stuck at the model building phase.
Would be highly grateful to you.
Thanks!

**Comment by JizhiziLi** [comment]
@Twice22  Hi there,

Thanks for showing interest in our project, for the reimplementation of paper Hattmatting, since they have not defined some part of the structure explicitly, I have used some customized blocks. The repo I have used to calculate the number of params is reference as here https://github.com/Lyken17/pytorch-OpCounter. 

Since the paper is under review now, we cannot release our code until then, maybe it is better to contact the original author about the params of their network, here is a reference to their repo: https://github.com/wukaoliu/CVPR2020-HAttMatting. Thanks!

Cheers,
Jizhizi Li

**Comment by Twice22** [comment]
Hey!

Thank you for your answer :). Actually I've seen other papers mentioning that they have also implemented it with a different number of parameters. Which is, as you mentioned, due to some lacks of details in the paper.

Anyway, I'll keep with my reimplementation :).

Wish you all the best for your submission 👍
